### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by devanshiwebiots**

### CODE REVIEW

File: src/Components/General/Dashboards/DefaultDashboard/Information/NotificationList.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <div className="flex-grow-1 ms-2">
+        <Link href={Routes.Users.UserProfile}>
+          <h5>
+            {name} <strong>{action}</strong> {detail}
+          </h5>
+          <span>{time}</span>
+        </Link>
+        {buttons && (
+          <>
+            <button className="btn btn-secondary mt-1 mb-1">Decline</button>
```
Comment (webiotsTanu) [inline src/Components/General/Dashboards/DefaultDashboard/Information/NotificationList.tsx]: why not reactstrap button?

File: src/Components/General/Dashboards/EcommerceDashboard/UserCountry.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                <DashboardCommonHeader title={Country} />
+                <CardBody>
+                    <div className="d-flex">
+                        <h2 className="me-2">216.459</h2>
+                        <span className="bg-light-success"><SVG iconId="up-arrow" /></span>
+                        <h6 className="font-success">+ 5.09%</h6>
+                    </div>
+                    <span>Increase than last month</span>
+                    <div className="jvector-map-height" id="world-map2">
+                        <VectorMap
```
Comment (webiotsTanu) [inline src/Components/General/Dashboards/EcommerceDashboard/UserCountry.tsx:23]: why vector map is used, is it used in other themes?

File: src/Components/General/Dashboards/DefaultDashboard/Information/NotificationList.tsx (hunk 1 above)
Comment (devanshiwebiots) [reply src/Components/General/Dashboards/DefaultDashboard/Information/NotificationList.tsx]: Done

File: src/Components/General/Dashboards/EcommerceDashboard/UserCountry.tsx (hunk 2 above)
Comment (devanshiwebiots) [reply src/Components/General/Dashboards/EcommerceDashboard/UserCountry.tsx:23]: It is used in Berlin next.I tried Using World map but design got broken and also vector map props were more convenient that's why used vector map

Comment (webiotsTanu) [reply src/Components/General/Dashboards/EcommerceDashboard/UserCountry.tsx:23]: okay

**Review by webiotsTanu — APPROVED** [review]

**Pull request merged**